src/include/ipc/channel_map.h

```diff
@@ -41,7 +41,7 @@ struct sof_ipc_channel_map {
 	uint32_t ch_mask;
 	uint32_t reserved;
 	int32_t ch_coeffs[0];
-} __packed;
+} __attribute__((packed));
```

Comment:
Can't we have a macro that does `#define __packed __attribute__((packed))` somewhere?

Reply:
@paulstelian97 we already have the macro, the problem is inconsistent usage and macro is also duplicated on Zephyr too.  We will have to live with some duplication until the port is done when we can clean up.